src/view/output.js

```diff
@@ -0,0 +1,31 @@
+import { FORMATTING_TYPE, GAME_VALUE } from '../constants/index.js';
+
+const output = (data, formattingType = '') => {
+  if (formattingType === '') {
+    return console.log(data);
+  }
+
+  if (formattingType === FORMATTING_TYPE.BOUGHT_LOTTOS) {
+    console.log(`${data.length}개를 구매했습니다.`);
+
+    return data.forEach((lotto) => {
+      console.log(`[${lotto.join(', ')}]`);
+    });
+  }
+
+  if (formattingType === FORMATTING_TYPE.WINNING_STATUS) {
+    console.log(`\n당첨 통계\n${'-'.repeat(20)}`);
+    console.log(`3개 일치 (${GAME_VALUE.PRIZE[4].toLocaleString()}원) - ${data.fifth}개
+4개 일치 (${GAME_VALUE.PRIZE[3].toLocaleString()}원) - ${data.fourth}개
+5개 일치 (${GAME_VALUE.PRIZE[2].toLocaleString()}원) - ${data.third}개
+5개 일치, 보너스 볼 일치 (${GAME_VALUE.PRIZE[1].toLocaleString()}원) - ${data.second}개
+6개 일치 (${GAME_VALUE.PRIZE[0].toLocaleString()}원) - ${data.first}개`);
```

Comment:
```suggestion
    console.log(`3개 일치 (${GAME_VALUE.PRIZE[4].toLocaleString()}원) - ${data.fifth}개`);
    console.log(`4개 일치 (${GAME_VALUE.PRIZE[3].toLocaleString()}원) - ${data.fourth}개`);
    console.log(`5개 일치 (${GAME_VALUE.PRIZE[2].toLocaleString()}원) - ${data.third}개`);
    console.log(`5개 일치, 보너스 볼 일치 (${GAME_VALUE.PRIZE[1].toLocaleString()}원) - ${data.second}개`);
    console.log(`6개 일치 (${GAME_VALUE.PRIZE[0].toLocaleString()}원) - ${data.first}개`);
```

어차피 하나하나 작성할거였으면 이렇게 작성했을 때 가독성이 더 좋지 않을까요?! (무조건 고치라는건 아닙니당!)

Reply:
말씀하신 방법을 사용하면 탬플릿 문자열을 출력하면서 가독성이 망가지는 것을 해결할 수 있을 것 같아요. 그렇지만 저는 다른 방법을 시도해 보았어요.

`console.log()` 가 여러 줄에 걸쳐 반복해서 나타나는 점을 고치고 싶어서, 별도의 라이브러리를 만들어서 사용했는데, 이 방법은 어떻게 생각하시나요? 또한, 현업에서 위의 예시와 같이 여러 줄의 문자열을 출력해야 하는 상황에서 헤일리는 `console.log()` 을 각 줄마다 넣는 방법을 사용하시나요?

```JavaScript
const Trimmer = {
  /**
   * 탬플릿 형태의 문자열을 코드 내에서 들여쓰기를 사용하여 나타낼 수 있도록 고안된 함수입니다.
   * 첫 줄을 제거하고, 각 줄마다 앞부분의 불필요한 공백을 제거한 후 리턴합니다.
   *
   * @param {string} message - 다듬을 문자열
   * @returns {string}
   */

  trimTemplate(message) {
    return message
      .replace(/^\n|\n$/g, '')
      .split('\n')
      .map((line) => line.trim())
      .join('\n');
  },
};

export default Trimmer;
```

```JavaScript
console.log(
  Trimmer.trimTemplate(`
    \n당첨 통계\n${'-'.repeat(20)}
    3개 일치 (${GAME_VALUE.PRIZE[4].toLocaleString()}원) - ${data.fifth}개
    4개 일치 (${GAME_VALUE.PRIZE[3].toLocaleString()}원) - ${data.fourth}개
    5개 일치 (${GAME_VALUE.PRIZE[2].toLocaleString()}원) - ${data.third}개
    5개 일치, 보너스 볼 일치 (${GAME_VALUE.PRIZE[1].toLocaleString()}원) - ${data.second}개
    6개 일치 (${GAME_VALUE.PRIZE[0].toLocaleString()}원) - ${data.first}개
  `)
);
```

Reply:
추가로, `constants/index.js` 에 상수 형태로 저장했습니다!
```JavaScript
PRIZE_RESULT: (gameData) =>
  Trimmer.trimTemplate(
    `
      당첨 통계
      ${'-'.repeat(20)}
      3개 일치 (${GAME_VALUE.PRIZE[4].toLocaleString()}원) - ${gameData.fifth}개
      4개 일치 (${GAME_VALUE.PRIZE[3].toLocaleString()}원) - ${gameData.fourth}개
      5개 일치 (${GAME_VALUE.PRIZE[2].toLocaleString()}원) - ${gameData.third}개
      5개 일치, 보너스 볼 일치 (${GAME_VALUE.PRIZE[1].toLocaleString()}원) - ${gameData.second}개
      6개 일치 (${GAME_VALUE.PRIZE[0].toLocaleString()}원) - ${gameData.first}개`
    ),
```

Reply:
오!!! 저보다 한단계 더 나아가계시는군요! 👍